app/api/api.rb

```diff
@@ -0,0 +1,5 @@
+class Api < Grape::API
+  prefix "api"
+  format :json
+  mount V1::Healthcheck
```

Comment:
I think a path based version would be better, for consistency with the other API if nothing else?

https://github.com/ruby-grape/grape?tab=readme-ov-file#path

Reply:
I've added the versioning as a separate commit as it was too complex to squash into the existing commits!